distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -42,3 +42,13 @@ grant codeBase "file:/catalog-core-directorymonitor/org.apache.camel.camel-core/
     // permission java.io.FilePermission "/test/some/monitored/directory", "read";
     // permission java.io.FilePermission "/test/some/monitored/directory${/}-", "read, write";
 }
+
+grant codeBase "file:/pax-logging-log4j2" {
```

Comment:
❓ Should we comment the whole block?

Reply:
I agree and made it look similar to the one above it